### PR TITLE
fix: add delay for shared note true unmount

### PR DESCRIPTION
**Pull request by Tainan404**
### What does this PR do?

Add a delay to truely unmount the shared notes, when the shared notes is unmounted the connection with the etherpad is closed ans takes a few second to reconnect, delaying it we hold the connection when the user is navigating through other panels (chat, poll and etc)

### Motivation

Remove the speed bump though the navigation to another panels.

### More
2.6

https://user-images.githubusercontent.com/15806883/204860716-4289a01c-4cc8-48d0-9530-9958892a16db.mp4

PR:

https://user-images.githubusercontent.com/15806883/204860813-ac7e12ff-8780-4eec-926b-221ec0ed3db0.mp4

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/notes/component.jsx [hunk 1]
```diff
@@ -58,18 +61,31 @@ const Notes = ({
...
     position: 'absolute',
     ...sharedNotesOutput,
   } : {};
-  const isHidden = isOnMediaArea && (style.width === 0 || style.height === 0);
+  const isHidden = (isOnMediaArea && (style.width === 0 || style.height === 0))
+                   || !isToSharedNotesBeShow;
 
   if (isHidden) style.padding = 0;
-
+  console.log();
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/ui/components/notes/component.jsx:78]:
```suggestion
```

File: bigbluebutton-html5/imports/ui/components/notes/component.jsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ import NotesDropdown from '/imports/ui/components/notes/notes-dropdown/container
 
 const CHAT_CONFIG = Meteor.settings.public.chat;
 const PUBLIC_CHAT_ID = CHAT_CONFIG.public_id;
-
+const DELAY_UNMOUNT_SHARED_NOTES = Meteor.settings.public.app.delayForUnmountOfSharedNote;
+console.log(DELAY_UNMOUNT_SHARED_NOTES);
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/ui/components/notes/component.jsx:16]:
```suggestion
```

**Review by ramonlsouza — CHANGES_REQUESTED** [review]
"write closed captions" and "pin notes onto whiteboard" features are not working correctly

[Screencast from 2023-01-09 14-53-54.webm](https://user-images.githubusercontent.com/3728706/211375824-18d8c7e8-f8aa-4c67-ba04-1408c6d80043.webm)

[Screencast from 2023-01-09 14-57-32.webm](https://user-images.githubusercontent.com/3728706/211376202-53fef49e-fd86-4155-95e4-07c9eda6c95a.webm)

**Review by ramonlsouza — APPROVED** [review]

File: bigbluebutton-html5/private/config/settings.yml [hunk 3]
```diff
@@ -33,6 +33,7 @@ public:
     copyright: '©2023 BigBlueButton Inc.'
     html5ClientBuild: HTML5_CLIENT_VERSION
     helpLink: https://bigbluebutton.org/html5/
+    delayForUnmountOfSharedNote: 120000
```
Comment (antobinary) [inline bigbluebutton-html5/private/config/settings.yml:36]: Could you please add an inline comment stating what this is (and what it means to reduce/increase the value; also -- milliseconds?)

**Pull request merged**